### PR TITLE
Adding configuration value for name

### PR DESCRIPTION
**Pull request by atxbyea**
was shown in examples, but not described

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/nut.markdown [hunk 1]
```diff
@@ -41,6 +41,11 @@ sensor:
     required: false
     default: 3493
     type: integer
+  name:
+    description: Custom name of the sensor
+    required: false
+    default: none
```
Comment (frenck) [inline source/_integrations/nut.markdown]: It actually has a default.

```suggestion
    default: NUT UPS
```

**Review by frenck — CHANGES_REQUESTED** [review]
Small suggestion @atxbyea, could you take a look?

**Comment by atxbyea** [comment]
Looks good to me @frenck

**Review by frenck — APPROVED** [review]
Thanks, @atxbyea 👍

**Pull request merged**